build.sbt

```diff
@@ -82,12 +81,12 @@ val catsVersion = "2.8.0"
 val catsEffectVersion = "3.3.14"
 val scodecBitsVersion = "1.1.34"
 val munitVersion = "0.7.29"
-val munitCEVersion = "1.0.7"
-val disciplineMUnitVersion = "1.0.9"
+val munitCEVersion = "2.0.0-M3"
```

Comment:
Upgrades were necessary because I ran into linker issues where certain functions weren't defined for scala native.